7.23 astro release notes (#3961)
astronomer/docs: astro/release-notes.md

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## July 23, 2024

### Run your first DAG on Astro with GitHub Integration

Now, when you first set up your Astro account, you can choose to connect your GitHub account and run a sample Astro project template and DAG. Previously, to run your first DAG on Astro, you needed to either download and use the Astro CLI or work with GitHub Actions. Now, the onboarding process allows you to customize your first experience with Astro to focus on your use case, whether that's Business Opeartions, Generative AI, or learning Airflow, and set up a GitHub Integration to quickly clone and deploy the example DAGs. Try it now with [Start Astro](https://cloud.astronomer.io/start-astro).

> ### Work with dbt projects on Astro

Author: @iancmoritz - are we launching in time for this to go into release notes, or should it wait for next week since it's going out on Wednesday?

:::privatepreview
:::

### Additional improvements

- View your Astro resource use with [Organization Dashboards](https://www.astronomer.io/docs/astro/organization-dashboard).

> - Raised the maximum worker node and executor sizes in Astro Hosted Dedicated and Astro Hybrid from 100 to 200 GiB, to allow large clusters to scale up further.

Reviewer: No change here. Max is still 100.

<!-- ALL LINKS TO INTERNAL DOCS MUST BE COMPLETE URLS INCLUDING HTTPS. Otherwise the links will break in RSS. -->

## July 23, 2024

### Run your first DAG on Astro with GitHub Integration

> Now, when you first set up your Astro account, you can choose to connect your GitHub account and run a sample Astro project template and DAG. Previously, to run your first DAG on Astro, you needed to either download and use the Astro CLI or work with GitHub Actions. Now, the onboarding process allows you to customize your first experience with Astro to focus on your use case, whether that's Business Opeartions, Generative AI, or learning Airflow, and set up a GitHub Integration to quickly clone and deploy the example DAGs. Try it now with [Start Astro](https://cloud.astronomer.io/start-astro).

Reviewer 2: ```suggestion
Now, when you first set up your Astro account, you can choose to connect your GitHub account and run a sample Astro project template and DAG. Previously, to run your first DAG on Astro, you needed to either download and use the Astro CLI or work with GitHub Actions. Now, the onboarding process allows you to customize your first experience with Astro to focus on your use case, whether that's Business Operations, Generative AI, or learning Airflow, and set up a GitHub Integration to quickly clone and deploy the example DAGs. Try it now with [Start Astro](https://cloud.astronomer.io/start-astro).
```

### Work with dbt projects on Astro

<HostedBadge/>

:::privatepreview
:::

> You can now deploy dbt code to Astro quickly and independently from your DAG or image deploys. Data build tool (dbt) core is an open source tool for data transformation that uses SQL, instead of transforming your data with Python in your DAGs. You can also use both Cosmos, an open source tool that integrates dbt models into Airflow and treats them like DAGs, and dbt deploys on Astro to have unparalleled visibility into your dbt tasks with the Airflow UI and a streamlined code deploy process.

Reviewer 2: ```suggestion
You can now deploy dbt code to Astro quickly and independently of your DAG or image deploys. Data build tool (dbt) core is an open source tool for data transformation that uses SQL to transform your data instead of Python in your DAGs. You can also use both Cosmos, an open source tool that integrates dbt models into Airflow and treats them like DAGs, and dbt deploys on Astro to have unparalleled visibility into your dbt tasks with the Airflow UI and a streamlined code deploy process.
```

Now, when you first set up your Astro account, you can choose to connect your GitHub account and run a sample Astro project template and DAG. Previously, to run your first DAG on Astro, you needed to either download and use the Astro CLI or work with GitHub Actions. Now, the onboarding process allows you to customize your first experience with Astro to focus on your use case, whether that's Business Operations, Generative AI, or learning Airflow, and set up a GitHub Integration to quickly clone and deploy the example DAGs. Try it now with [Start Astro](https://cloud.astronomer.io/start-astro).

### Work with dbt projects on Astro

<HostedBadge/>

:::privatepreview
:::

You can now deploy dbt code to Astro quickly and independently of your DAG or image deploys. Data build tool (dbt) core is an open source tool for data transformation that uses SQL to transform your data instead of Python in your DAGs. You can also use both Cosmos, an open source tool that integrates dbt models into Airflow and treats them like DAGs, and dbt deploys on Astro to have unparalleled visibility into your dbt tasks with the Airflow UI and a streamlined code deploy process.

> See [dbt project deploys](https://www.astronomer.io/docs/astro/deploy-dbt-project) for more information about how to get started.

Author: ```suggestion
See [Deploy dbt projects](https://www.astronomer.io/docs/astro/deploy-dbt-project) for more information about how to get started.
```

### Additional improvements

> - View your Astro resource use with [Organization Dashboards](https://www.astronomer.io/docs/astro/organization-dashboard).

Author: ```suggestion
```

> - Raised the maximum worker node and executor sizes in Astro Hosted Dedicated and Astro Hybrid from 100 to 200 GiB, to allow large clusters to scale up further.

Author: ```suggestion
```

> - Added a new **Extra Large** sized Scheduler.

Author: ```suggestion
```
